pgjdbc/src/main/java/org/postgresql/PGProperty.java

```diff
@@ -375,14 +375,14 @@
       "extended", "extendedForPrepared", "extendedCacheEveryting", "simple"),
```

Comment:
Should be fixed the property too? "extendedCacheEveryting" => "extendedCacheEverything"